ethbloom/Cargo.toml

```diff
@@ -11,7 +11,8 @@ repository = "https://github.com/paritytech/primitives"
 [dependencies]
 tiny-keccak = "1.4"
 crunchy = { version = "0.1.6", features = ["limit_256"] }
-fixed-hash = { version = "0.2.1", path = "../fixed-hash", default-features = false }
+# TODO: remove `branch` when https://github.com/paritytech/parity-common/pull/12 lands
```

Comment:
Seems it landed already

Reply:
I also think it would be best to actually use crates.io version instead of git repo.

Reply:
> Seems it landed already

Good catch, will fix. 

> I also think it would be best to actually use crates.io version instead of git repo.

I agree with this (strongly) but I wanted to wait until this stuff lands in polkadot (this poorly named [PR](https://github.com/paritytech/polkadot/pull/297)) and the dust settles a little bit.